### PR TITLE
Added comments for pll_setting_t and pll_settings_str

### PR DESCRIPTION
**Pull request by ericxu233**
Addresses the issue of missing documentation for clReconfigurePLLIntelFPGA input parameter #111 .

### CODE REVIEW

File: include/acl_pll.h [hunk 1]
```diff
@@ -8,16 +8,19 @@
...
-  unsigned int freq_khz;
-  unsigned int m;
-  unsigned int n;
-  unsigned int k;
-  unsigned int c0;
-  unsigned int c1;
-  unsigned int r;
-  unsigned int cp;
-  unsigned int div;
+  unsigned int freq_khz; /* output frequency in kHZ */
```
Comment (pcolberg) [inline include/acl_pll.h:15]:
```suggestion
  unsigned int freq_khz; /* output frequency in kHz */
```

File: include/acl_pll.h [hunk 2]
```diff
@@ -8,16 +8,19 @@
...
+  unsigned int m;        /* multiplier factor */
+  unsigned int n;        /* 1st divider factor */
+  unsigned int k;        /* 2nd divider factor */
+  unsigned int c0;       /* output divider for kernal clock */
+  unsigned int c1;       /* output divider for double kernal clock */
```
Comment (pcolberg) [inline include/acl_pll.h:20]:
```suggestion
  unsigned int c0;       /* output divider for kernel clock */
  unsigned int c1;       /* output divider for double kernel clock */
```

**Review by pcolberg — CHANGES_REQUESTED** [review]

**Comment by pcolberg** [comment]
Thanks @ericxu233, looks good! I have corrected two minor spelling mistakes, and will merge after CI has passed.

**Review by pcolberg — APPROVED** [review]

**Review by pcolberg — APPROVED** [review]

**Comment by ericxu233** [comment]
Thanks @pcolberg ! I am really happy to have submitted my first commit on my very first open source project contribution!

**Pull request merged**